pkg/controllers/cloudentityselector/controller.go

```diff
@@ -138,13 +136,13 @@ func (r *CloudEntitySelectorReconciler) processCreateOrUpdate(selector *crdv1alp
 		Name:      selector.Spec.AccountName,
 	}
 	r.selectorToAccountMap[*selectorNamespacedName] = *accountNamespacedName
-	retry, err := r.AccManager.AddResourceFiltersToAccount(accountNamespacedName, selectorNamespacedName,
+	// No errors can be retried for a CES.
+	_, err := r.AccManager.AddResourceFiltersToAccount(accountNamespacedName, selectorNamespacedName,
```

Comment:
@Anandkumar26 Can you confirm this?

Reply:
Yes. The error while processing CES can only occur if an account is either not initialised successfully or account is in a bad state.  I don't think there is any benefit in retrying the errors forever. 

So we capture the error and store in a CES status field. (which can be replayed when account goes from an uninitialised to initialised state OR it would need a user intervention)